docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1422,6 +1422,21 @@
                     </td>
                 </tr>
 
+                <tr>
+                    <td class="property" id="stores-rocksdb-log-file-size">stores.<span class="store">store-name</span>.<br>rocksdb.max.log.file.size.bytes</td>
+                    <td class="default">67108864</td>
+                    <td class="description">
+                        The maximum size in bytes of the RocksDB LOG file before it is rotated.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="stores-rocksdb-num-log-files">stores.<span class="store">store-name</span>.<br>rocksdb.keep.log.file.num</td>
```

Comment:
Yeah, better to mirror RocksDB configuration as-is for these. I can rename the html id for consistency if you'd like.